### DISCUSSION TITLE
Question about organization policies

### DISCUSSION THREAD

**Discussion by mariuskimmina** [ctreminiom/go-atlassian#249] [General]
Hey, 

When getting policies from https://docs.go-atlassian.io/atlassian-admin-cloud/organization/policy the page is strictly talking about authentication policies but as far as I can tell it doesn't seem possible to get those via the atlassian API, see https://jira.atlassian.com/browse/ACCESS-1044. 
I currently only receive data residency policies when using `admin.Organization.Policy.Gets(context.Background(), orgId, "", "")`.

Is the documentation here misleading or should it actually be possible to get those authentication policies?

**Comment by ctreminiom** [comment]
Hi @mariuskimmina, I'm going to test it on the side and I'll reply you in a couple of days.
